Core/Security/X509Chain.py

```diff
@@ -153,6 +158,14 @@ def __getProxyExtensionList( self, diracGroup = False ):
                                           'critical, digitalSignature, keyEncipherment, dataEncipherment' ) )
     if diracGroup and type( diracGroup ) in self.__validExtensionValueTypes:
       extList.append( crypto.X509Extension( 'diracGroup', diracGroup ) )
+    if rfc or rfcLimited:
+      blob = [ [ "1.3.6.1.5.5.7.21.1" ] ] if not rfcLimited else [ [ "1.3.6.1.4.1.3536.1.1.1.9" ] ]
```

Comment:
That's nothing. Check out the beauty that the ASN1 structure a certificate is:

https://github.com/DIRACGrid/DIRAC/pull/2272/files#diff-7d2aac6b2e0163417e13f662e1e2ae88R176